gpu4pyscf/drivers/dft_driver.py

```diff
@@ -231,6 +233,20 @@ def run_dft(mol_name, config, charge=None, spin=0):
             h_dft = h.kernel().transpose([0,2,1,3]).reshape([3*natm, 3*natm])
             hess_time = time.time() - start_time
             print(f'compute time for hessian: {hess_time:.3f} s')
+
+            if with_thermo:
+                # harmonic analysis
+                start_time = time.time()
+                normal_mode = thermo.harmonic_analysis(mol, h.kernel())
```

Comment:
@liuyu-chem1996 `h.kernel()` is executed twice. It is expensive. Simply reuse `h_dft`.